src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -143,6 +143,30 @@ case class SubmissionReport(
   notstarted: Seq[SubmissionValidationEntityInputs]
 )
 
+case class CallMetadata(
+  inputs: Map[String, String],
+  status: String,
+  outputs: Option[Map[String, String]],
+  start: Option[DateTime],
+  end: Option[DateTime],
+  jobid: Option[String],
+  rc: Option[Int],
```

Comment:
I'm totally amenable to having it changed on the Cromwell side, although CallMetadataVoetesque has a nice ring to it. As it is right now someone wouldn't know if we meant it was the return code or a tasty cola beverage.

@mcovarr I'm imagining this is something which could be changed trivially on our side as we finish off the ticket?